setup.py

```diff
@@ -1,6 +1,18 @@
 from setuptools import setup
 
 
+INSTALL_REQUIRES = [
+    "attrs >= 15.2.0",
+    "okonomiyaki == 0.14.0",
+    "six >= 1.9.0"
+]
+
+EXTRAS_REQUIRE = {
+    ':python_version=="2.7"': [
```

Comment:
This doesn't seem to work. Should I have to do anything other than `pip install -e .` ?